lib/Quantization/Quantization.cpp

```diff
@@ -163,10 +163,11 @@ class FunctionQuantizer : public FunctionConverter {
   Node *createConversion(Function &function, NodeValue &val,
                          TypeRef destTy) override {
     if (destTy->isQuantizedType()) {
-      return function_.createQuantize("quantize", val, destTy);
+      return function.createQuantize("quantize", val, destTy);
```

Comment:
For this one, I'm guessing the `function` parameter was intended but I'm not sure.

Reply:
hm, let me see :-) interesting find here.

Reply:
Nice catch!

Is it possible to add a test to check this?

Reply:
Also, this seems to be a candidate to be a static method.

Reply:
@jfix71 I'm not actually sure how to test this, I'm not very familiar with this part of the code yet.
@rdzhabarov do you know if we could actually just remove the function parameter to this createConversion() and use `function_` instead? It seems like the whole functionConverter thing only works on one Function at a time so I don't know if it really makes sense to be able to pass in another Function ever. As for static method, I think you can't have a static method that also overridden.

Reply:
You can easily test if you make that static ;)

You could remove `function` param, but it's not going to be as generic as it's now.
I'd vote to make that static, pass function and add a unit test.